Source/WebCore/css/CSSCounterStyleDescriptors.cpp

```diff
@@ -251,54 +303,243 @@ bool CSSCounterStyleDescriptors::areSymbolsValidForSystem() const
 
 bool CSSCounterStyleDescriptors::isValid() const
 {
-    return areSymbolsValidForSystem();
+    return areSymbolsValidForSystem(m_system, m_symbols, m_additiveSymbols);
+}
+
+bool CSSCounterStyleDescriptors::setName(CSSCounterStyleDescriptors::Name name)
+{
+    if (name.isNull() || m_name == name)
+        return false;
+    m_name = WTFMove(name);
+    return true;
+}
+
+bool CSSCounterStyleDescriptors::setSystemData(CSSCounterStyleDescriptors::SystemData systemData)
+{
+    if (m_extendsName == systemData.first && m_fixedSystemFirstSymbolValue == systemData.second)
+        return false;
+    m_extendsName = systemData.first;
+    m_fixedSystemFirstSymbolValue = systemData.second;
+    return true;
 }
```

Comment:
I don't think the return bools in these functions are used. The code could be simplified so these are just simple setters.